### DISCUSSION TITLE
Near Future Launch Vehicles Tanks / Tweakscale

### DISCUSSION THREAD

**Discussion by mikken40** [TweakScale/TweakScale#326] [Support]
Hello,

I use Realism Overhaul and RealFuels.
I have created a patch for the NFLV tanks so that I can also use RealFuels with them. That works great so far.
But the problem is that I can no longer scale these tanks with Tweakscale?
What could be the reason for this? I have already read that RealFuels in conjunction with Tweakscale causes problems for some. And it has been deactivated in the RealFuels settings.
However, I would like to test this myself and cannot find the setting to reactivate Tweakscale for RealFuels.

I thank you in advance for any comment and hope it is described clearly ;)

LG Mike

**Comment by Lisias** [comment]
This is a duplicate for https://github.com/TweakScale/Companion_FuelSwitches/issues/7
